config/environments/production.rb

```diff
@@ -75,18 +80,23 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation cannot be found).
-  config.i18n.fallbacks = [I18n.default_locale]
+  config.i18n.fallbacks = true
```

Comment:
for anyone wondering about this, this is what <https://guides.rubyonrails.org/configuring.html> says:

> You can set the option to true for using default locale as fallback